src/SettingsPage/View.php

```diff
@@ -59,6 +62,35 @@ public function add_options_page() {
 	 */
 	public function render_page() {
 
+		$this->register_scripts_styles();
+
+		$tabs = [
+			new AdminBar(),
+			new Dashboard(),
+			new Menu()
```

Comment:
Here we need a hook, so that we can add custom areas, third plugin should also add tabs with settings.

Reply:
Added a filter hook for the tabs array.